src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ConditionSpecificLogic.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
+
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.AdditionalCondition
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.AdditionalConditionData
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.Licence
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+val LONG_DATE_FORMATTER = DateTimeFormatter.ofPattern("EEEE dd MMMM yyyy")
+const val CONDITION_CODE_FOR_14B = "524f2fd6-ad53-47dd-8edc-2161d3dd2ed4"
+
+fun AdditionalCondition.getInitialData(): List<AdditionalConditionData> = when (conditionCode) {
+  CONDITION_CODE_FOR_14B -> createAdditionalConditionDataFor14b()
+  else -> emptyList()
+}
+
+private fun AdditionalCondition.createAdditionalConditionDataFor14b() = listOf(
+  AdditionalConditionData(
+    dataSequence = 0,
+    dataField = "endDate",
+    dataValue = licence.getElectronicMonitoringEndDate(),
+    additionalCondition = this
+  ),
+  AdditionalConditionData(
+    dataSequence = 1,
+    dataField = "infoInputReviewed",
+    dataValue = "false",
+    additionalCondition = this
+  )
+)
+
+fun Licence.getElectronicMonitoringEndDate(): String {
+  val date = when {
+    isLicenceExpiryLessThan12Months() -> licenceExpiryDate!!
+    actualReleaseDate != null -> actualReleaseDate!!.plusYears(1)
```

Comment:
Super minor but you could check for the class Type if you want to strict:
`actualReleaseDate is LocalDate -> actualReleaseDate!!.plusYears(1)`

The smart cast files so I can't see an easy way to not use !!